x-pack/package.json

```diff
@@ -214,9 +248,12 @@
     "pluralize": "3.1.0",
     "pngjs": "3.3.1",
     "polished": "^1.9.2",
+    "popper.js": "^1.14.3",
     "prop-types": "^15.6.0",
+    "proper-lockfile": "^3.0.2",
     "puid": "1.0.5",
     "puppeteer-core": "^1.7.0",
+    "query-string": "^6.1.0",
```

Comment:
No need for this package, please just use the native `querystring` module (which works in the browser too)